static/public/index.html

```diff
@@ -6,6 +6,359 @@
+            <svg width="12" height="8" viewBox="0 0 12 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path d="M10.293 0.292999L5.99997 4.586L1.70697 0.292999L0.292969 1.707L5.99997 7.414L11.707 1.707L10.293 0.292999Z" fill="white"/>
+              </svg>
+          </button>
+          <button class="btn--primary-sm m-4 gap-xs">
+            Small Btn
+            <svg width="12" height="8" viewBox="0 0 12 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path d="M10.293 0.292999L5.99997 4.586L1.70697 0.292999L0.292969 1.707L5.99997 7.414L11.707 1.707L10.293 0.292999Z" fill="white"/>
+              </svg>
+          </button>
+        </div>
+      </div>
+      <!-- Buttons with icons -->
+      <div class="flex flex-col">
+        <h1 class="text-hero-large-bold mb-4">Buttons with icons :</h1>
+        <div class="flex flex-row gap-16">
+          <button class="btn--primary rounded-full text-md w-14 h-14">
+            <svg width="16" height="10" viewBox="0 0 16 10" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path d="M13.724 0.390625L7.99996 6.11463L2.27596 0.390625L0.390625 2.27596L7.99996 9.88529L15.6093 2.27596L13.724 0.390625Z" fill="white"/>
+              </svg>
+          </button>
+          <button class="btn--primary rounded-full text-base w-10 h-10">
+            <svg width="12" height="8" viewBox="0 0 12 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path d="M10.293 0.292969L5.99997 4.58597L1.70697 0.292969L0.292969 1.70697L5.99997 7.41397L11.707 1.70697L10.293 0.292969Z" fill="white"/>
+              </svg>              
+          </button>
+          <button class="btn--primary rounded-full text-sm w-8 h-8">
+            <svg width="8" height="5" viewBox="0 0 8 5" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path d="M6.86198 0.195312L3.99998 3.05731L1.13798 0.195312L0.195312 1.13798L3.99998 4.94265L7.80465 1.13798L6.86198 0.195312Z" fill="white"/>
+              </svg>              
+          </button>
+        </div>
+      </div>
+      <!-- Sliders -->
+      <div class="flex flex-col">
+        <h1 class="text-hero-large-bold mb-4">Sliders :</h1>
+        <div class="flex flex-row gap-10">
+          <p class="text-title">Complete</p>
+          <div class="h-3  w-52 rounded-xsm overflow-hidden bg-opacity-90">
+            <div class="bg-success w-full h-1/4"></div>
+          </div>
+          <p class="text-title">Progress</p>
+          <div class="h-3  w-52 rounded-xsm overflow-hidden bg-opacity-90">
+            <div class="bg-primary w-40 h-1/4"></div>
+          </div>
+          <div class="h-3 w-52 rounded-xsm overflow-hidden bg-opacity-90">
+            <div class="bg-primary w-1 h-1"></div>
+          </div>
+        </div>
+        <h1 class="text-2xl md:text-4xl font-medium mb-4">Vote meter :</h1>
+        <div class="flex flex-row gap-44">
+          <div class=" bg-white rounded-xsm overflow-hidden">
```

Comment:
- border-radius should be on the slider bar not in the div container with `rounded-full`
  - but I recommend putting it in both (bar and container) as would be obvious on the next part when start building molecules .
```suggestion
          <div class=" bg-white overflow-hidden">
            <div class="bg-primary-shd5 rounded-full w-2 h-80 p-1"></div>
          </div>
```